### PR TITLE
CU-th3fhq: UX fix for exceeding minion names

### PR DESCRIPTION
**Pull request by sunguru98**
The new minions' section (which is present over the DAO page), is shown as follows 

Screenshot 2021-06-17 at 10 51 48 PM

The Copy icon has been swept to right and applied address removal if the minion name is greater than 15 chars .. 
As usual, I would always be hearing for comments :)

### CODE REVIEW

**Comment by scottrepreneur** [comment]
Task linked: [CU-th3fhq clean up Minion List names](https://app.clickup.com/t/th3fhq)

**Comment by scottrepreneur** [comment]
Task linked: [CU-n2z9pw bg opacity slider](https://app.clickup.com/t/n2z9pw)

File: src/components/minionList.jsx [hunk 1]
```diff
@@ -31,6 +32,40 @@ const MinionList = () => {
...
+    if (!minionType) return 'minon';
+    switch (minionType) {
+      case MINION_TYPES.SUPERFLUID:
+        return {
+          minionUrlType: 'superfluid-minon',
+          badgeColor: 'green',
+          badgeTextColor: 'white',
+          badgeName: 'SF',
+        };
+      case MINION_TYPES.ENS:
```
Comment (skuhlmann) [inline src/components/minionList.jsx]: We don't have an ens minion - so let's remove this for now.

**Review by scottrepreneur — APPROVED** [review]

File: src/components/minionList.jsx (hunk 1 above)
Comment (sunguru98) [reply src/components/minionList.jsx]: Sure thing. On it.

**Review by scottrepreneur — APPROVED** [review]

**Pull request merged**